sssd_test_framework/roles/ipa.py

```diff
@@ -1026,7 +1028,15 @@ def passkey_add(self, passkey_mapping: str) -> IPAUser:
         self._exec("add-passkey", [passkey_mapping])
         return self
 
-    def passkey_add_register(
+    def passkey_add_register(self, **kwargs) -> str:
+        """wrapper for *_passkey_add_register methods"""
```

Comment:
Apparently tox is unable to parse correctly the * character. You can replace it by
```
"""wrapper for \\*_passkey_add_register methods"""
```

or remove it altogether.

There are other instances of this error, so replace all of them together.